qa/integration-tests/src/test/java/io/camunda/it/auth/TenantAuthorizationIT.java

```diff
@@ -0,0 +1,215 @@
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.it.auth;
+
+import static io.camunda.client.protocol.rest.PermissionTypeEnum.CREATE;
+import static io.camunda.client.protocol.rest.PermissionTypeEnum.READ;
+import static io.camunda.client.protocol.rest.ResourceTypeEnum.TENANT;
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.camunda.application.Profile;
+import io.camunda.client.CamundaClient;
+import io.camunda.it.utils.BrokerITInvocationProvider;
+import io.camunda.it.utils.CamundaClientTestFactory.Authenticated;
+import io.camunda.it.utils.CamundaClientTestFactory.Permissions;
+import io.camunda.it.utils.CamundaClientTestFactory.User;
+import io.camunda.zeebe.util.Either;
+import io.camunda.zeebe.util.collection.Tuple;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpRequest.BodyPublishers;
+import java.net.http.HttpResponse;
+import java.time.Duration;
+import java.util.Base64;
+import java.util.List;
+import java.util.UUID;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.AutoClose;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.springframework.http.HttpStatus;
+
+@TestInstance(Lifecycle.PER_CLASS)
+class TenantAuthorizationIT {
+
+  public static final ObjectMapper OBJECT_MAPPER =
+      new ObjectMapper().configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+  public static final String PASSWORD = "password";
+  private static final String ADMIN = "admin";
+  private static final String RESTRICTED = "restricted-user";
+  private static final String UNAUTHORIZED = "unauthorized-user";
+  private static final User ADMIN_USER =
+      new User(
+          ADMIN,
+          PASSWORD,
+          List.of(
+              new Permissions(TENANT, CREATE, List.of("*")),
+              new Permissions(TENANT, READ, List.of("*"))));
+  private static final User RESTRICTED_USER =
+      new User(
+          RESTRICTED,
+          PASSWORD,
+          List.of(new Permissions(TENANT, READ, List.of("tenant1", "tenant2"))));
+  private static final User UNAUTHORIZED_USER = new User(UNAUTHORIZED, PASSWORD, List.of());
+
+  @RegisterExtension
+  static final BrokerITInvocationProvider PROVIDER =
+      new BrokerITInvocationProvider()
+          .withoutRdbmsExporter()
+          .withAdditionalProfiles(Profile.AUTH_BASIC)
+          .withAuthorizationsEnabled()
+          .withUsers(ADMIN_USER, RESTRICTED_USER, UNAUTHORIZED_USER);
```

Comment:
Hey @remcowesterhoud, from similar tests, I think the issue starts here. This method eventually calls [this code](https://github.com/camunda/camunda/blob/main/zeebe/qa/util/src/main/java/io/camunda/zeebe/it/util/AuthorizationsUtil.java#L82-L96) to create permissions. However, the `AuthorizationAddPermissionsProcessor` currently doesn't work correctly so no permissions can be added. The solution is to switch to the new java client `CreateAuthorizationCommand`.

However, the create authorization feature isn't fully operational. I'm fixing some final things with [this PR](https://github.com/camunda/camunda/pull/27377), and I think that one will also unblock your test.